integration-cli/docker_api_network_test.go

```diff
@@ -17,15 +17,6 @@ import (
 	"github.com/go-check/check"
 )
 
-func (s *DockerSuite) TestAPINetworkGetDefaults(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	// By default docker daemon creates 3 networks. check if they are present
-	defaults := []string{"bridge", "host", "none"}
-	for _, nn := range defaults {
-		c.Assert(isNetworkAvailable(c, nn), checker.Equals, true)
-	}
-}
-
 func (s *DockerSuite) TestAPINetworkCreateDelete(c *check.C) {
```

Comment:
Perhaps would be good to combine a couple in a single PR; looks like there's some "easy ones" to migrate